test/integration/kubernetes_deploy_test.rb

```diff
@@ -213,4 +213,28 @@ def test_should_raise_if_required_binding_not_present
       deploy_fixtures('collection-with-erb', subset: ["conf_map.yml.erb"])
     end
   end
+
+  def test_long_running_deployment
+    2.times do
+      deploy_fixtures('long-running')
+    end
+
+    pods = kubeclient.get_pods(namespace: @namespace, label_selector: 'name=jobs,app=fixtures')
+    assert_equal 4, pods.size
+
+    count = count_by_revisions(pods)
+    assert_equal [2, 2], count.values
```

Comment:
We assert that both revisions are still running, after the latest one has been deployed.